packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -210,7 +217,8 @@ class Tooltip extends StatefulWidget {
   static bool dismissAllToolTips() {
     if (_openedToolTips.isNotEmpty) {
       // Avoid concurrent modification.
-      final List<_TooltipState> openedToolTips = List<_TooltipState>.from(_openedToolTips);
+      final List<_TooltipState> openedToolTips =
```

Comment:
Please revert all these unrelated code formatting changes. The flutter repo does not use dartfmt, see also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting

Reply:
(Here and elsewhere)